### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Tag and pre-deploy consignment export
```
Comment (TomJKing) [inline .github/workflows/release.yml:1]: Is this the correct name for this workflow?

File: .github/workflows/run.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            mkdir -p scripts-pages/docs/performance/${{ github.run_id }}/
+            cp report/* scripts-pages/docs/performance/${{ github.run_id }}/
+            cd scripts-pages
+            git add docs
+            git commit -m "Add performance checks ${{ github.run_id }}"
+            git push
+      - uses: nationalarchives/tdr-github-actions/.github/actions/slack-send@main
+        with:
+          message: "Backend check peformance run complete. See <https://nationalarchives.github.io/tdr-scripts/performance/${{ github.run_id }}/output.html|the report>"
+          slack-url: ${{ secrets.SLACK_WEBHOOK }}
```
Comment (TomJKing) [inline .github/workflows/run.yml:78]: Missing line break

File: terraform/root.tf [hunk 3]
```diff
@@ -6,6 +6,16 @@ provider "aws" {
   }
 }
 
+provider "aws" {
+  region = "us-east-1"
```
Comment (TomJKing) [inline terraform/root.tf:10]: Is that the correct region to be using?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just some minor comments and questions. Thanks

File: .github/scripts/copy-docker-images.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+aws ecr get-login-password --region eu-west-2 | docker login --username AWS --password-stdin $1.dkr.ecr.eu-west-2.amazonaws.com
+docker pull $1.dkr.ecr.eu-west-2.amazonaws.com/$3:intg
+docker tag $1.dkr.ecr.eu-west-2.amazonaws.com/$3:intg $2.dkr.ecr.eu-west-2.amazonaws.com/$3:sbox
+aws ecr get-login-password --region eu-west-2 | docker login --username AWS --password-stdin $2.dkr.ecr.eu-west-2.amazonaws.com
+docker push $2.dkr.ecr.eu-west-2.amazonaws.com/$3:sbox
```
[the remark this answers is not in the document]
Comment (MancunianSam) [reply .github/scripts/copy-docker-images.sh:6]: Ah, I added the line breaks and didn't push the changes.

File: .github/workflows/release.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/release.yml:1]: Yep absolutely. I've changed it anyway though :grin:

File: terraform/root.tf (hunk 3 above)
Comment (MancunianSam) [reply terraform/root.tf:10]: Terraform said I needed to add this while it sorted out its state file or something but I've just taken it out and it works fine without it.

**Review by TomJKing — DISMISSED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**